pages/index.tsx

```diff
@@ -92,18 +91,22 @@ const Home = ({ firstCompanies, initialResultsAmount }: HomeProps) => {
 };
 
 export const getStaticProps: GetStaticProps = async () => {
-  // Get data from database
-  const hoc = databaseHoc()(async (req) => {
-    const { companies } = await getCompanies(req.db, searchLimit);
-    return companies;
-  });
-  const firstCompanies = await hoc({} as NextRequestWithDb, {} as NextApiResponse);
-  return {
-    props: {
-      firstCompanies,
-      initialResultsAmount: searchLimit,
-    },
-  };
+  try {
+    const res = await fetch(`https://sertifikaattilukija.herokuapp.com/data?limit=${searchLimit}`);
+    const { data: companies } = await res.json();
+    return {
+      props: {
+        firstCompanies: companies,
+        initialResultAmount: searchLimit,
+      },
+    };
+  } catch (err) {
+    return {
+      props: {
+        error: err,
+      },
+    };
```

Comment:
Most likely error is a 404 somewhere in the process so I'd return
`    { notFound: true };
`
I'd also check that companies variable exists and is non empty

Reply:
I'll take that back, in the index page it shouldn't matter if we don't have prefetched companies so no need for 404s😅